client/components/data/domain-management/index.jsx

```diff
@@ -117,7 +88,6 @@ class DomainManagementData extends React.Component {
 						selectedDomainName={ this.props.selectedDomainName }
 						selectedSite={ selectedSite }
 						sitePlans={ this.props.sitePlans }
-						stores={ stores }
```

Comment:
It seems to me that the `StoreConnection` component lost all meaning by now and can be removed:
- doesn't maintain any store subscriptions
- the `isDataLoading` and `loadingPlaceholder` props are not used
- `getStateFromStores` merely passes props, with no local state left

Simply rendering `this.props.component` with the same props as `StoreConnection` had will do the same job. `domains` prop is passed conditionally, and `currentUser` prop is renamed to `user`, and all other props are simply passed through 1:1.

Reply:
Agreed! I was planning to clean that up in another PR, but don't mind doing it here. Cleaned up in the last 2 commits - https://github.com/Automattic/wp-calypso/pull/51809/commits/5ab49ae45c6edbf2c28e3048bab260a1d4295c7a and  https://github.com/Automattic/wp-calypso/pull/51809/commits/dfc1325ed9a5a74e168eece5271a87684dcb0a3d